### PR TITLE
add tutorial for workload-rebalancer

### PR DESCRIPTION
**Pull request by chaosi-zju**
**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

add tutorial for workload-rebalancer

**Which issue(s) this PR fixes**:
Fixes part of https://github.com/karmada-io/karmada/issues/4840

**Special notes for your reviewer**:

[quick-click](https://deploy-preview-612--karmada.netlify.app/docs/next/tutorials/workload-rebalancer)

### CODE REVIEW

File: docs/tutorials/workload-rebalancer.md [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+demo-deploy-1   3/3     3            3           3m18s
+
+$ kubectl --context member1 get po                                          
+NAME                             READY   STATUS    RESTARTS   AGE
+demo-deploy-1-784cd456bf-dv6xw   1/1     Running   0          3m18s
+demo-deploy-1-784cd456bf-fgjn7   1/1     Running   0          3m18s
+
+$ kubectl --context member2 get po                                          
+NAME                             READY   STATUS    RESTARTS   AGE
+demo-deploy-1-784cd456bf-856rf   1/1     Running   0          3m18s
```
Comment (RainbowMango) [inline docs/tutorials/workload-rebalancer.md:117]: Please use the command `karmadactl get deployments xxx`

File: docs/tutorials/workload-rebalancer.md [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+First prepare a Deployment named `demo-deploy-1`, you can create a new file `deployment.yaml` and content with the following:
+
+<details>
+<summary>deployment.yaml</summary>
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: demo-deploy-1
```
Comment (RainbowMango) [inline docs/tutorials/workload-rebalancer.md:46]:
```suggestion
  name: foo
```
or 
```suggestion
   name: foo-dep
```

File: docs/tutorials/workload-rebalancer.md [hunk 3]
```diff
@@ -0,0 +1,263 @@
...
+$ karmadactl --karmada-context=karmada-apiserver taint clusters member1 workload-rebalancer-test:NoExecute-
+cluster/member1 untainted
+```
+
+Removing the taint will not lead to replicas propagation changed for the reason of scheduling result inert,
+all replicas will keep in member2 cluster unchanged.
+
+### Step 3. apply a WorkloadRebalancer to trigger rescheduling.
+
+Assuming you want to trigger the rescheduling of above resources, you can create a new file `workload-rebalancer.yaml`
```
Comment (RainbowMango) [inline docs/tutorials/workload-rebalancer.md:157]:
```suggestion
In order to trigger the rescheduling of the above resources, you can create a new file `workload-rebalancer.yaml`
```

File: docs/tutorials/workload-rebalancer.md [hunk 4]
```diff
@@ -0,0 +1,263 @@
...
+
+```bash
+$ kubectl --context member1 get po                                          
+NAME                             READY   STATUS    RESTARTS   AGE
+demo-deploy-1-784cd456bf-82kt6   1/1     Running   0          89s
+demo-deploy-1-784cd456bf-k9fhl   1/1     Running   0          89s
+
+$ kubectl --context member2 get po
+NAME                             READY   STATUS    RESTARTS   AGE
+demo-deploy-1-784cd456bf-856rf   1/1     Running   0          9m23s
```
Comment (RainbowMango) [inline docs/tutorials/workload-rebalancer.md:223]: ditto. use `karmadactl` for better UX.

File: docs/tutorials/workload-rebalancer.md [hunk 5]
```diff
@@ -0,0 +1,263 @@
...
+
+```bash
+$ kubectl --context karmada-apiserver describe deployment demo-deploy-1
+...
+Events:
+  Type    Reason                  Age                From                                Message
+  ----    ------                  ----               ----                                -------
+  ...
+  Normal  ScheduleBindingSucceed  31s                default-scheduler                   Binding has been scheduled successfully. Result: {member2:2, member1:1}
+  Normal  GetDependenciesSucceed  31s                dependencies-distributor            Get dependencies([]) succeed.
```
Comment (RainbowMango) [inline docs/tutorials/workload-rebalancer.md:238]: @XiShanYongYe-Chang Please help to check why the `dependencies-distributor` would emit the event, note that the ResourceBinding's `PropagateDeps` is disabled.

Comment (chaosi-zju) [reply docs/tutorials/workload-rebalancer.md]: updated

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve

**Pull request merged**